### PR TITLE
Fixes retrieval of SSL Servlet request attributes.

### PR DESCRIPTION
**Pull request by sbordet**
Alternative to #9513.
Removed requestlog attributes that were never used.
Code cleanups to remove usage of deprecated methods.

### CODE REVIEW

File: jetty-ee10/jetty-ee10-proxy/src/test/java/org/eclipse/jetty/ee10/proxy/ClientAuthProxyTest.java [hunk 1]
```diff
@@ -117,7 +118,7 @@ private void startServer(Handler handler) throws Exception
         serverTLS.setSniRequired(false);
         serverTLS.setNeedClientAuth(true);
         // The KeyStore is also a TrustStore.
-        serverTLS.setKeyStorePath(MavenTestingUtils.getTestResourceFile("client_auth/server_keystore.p12").getAbsolutePath());
+        serverTLS.setKeyStorePath(MavenTestingUtils.getTestResourcePath("client_auth/server_keystore.p12").toAbsolutePath().toString());
```
Comment (joakime) [inline jetty-ee10/jetty-ee10-proxy/src/test/java/org/eclipse/jetty/ee10/proxy/ClientAuthProxyTest.java]: The `.toAbsolutePath()` part is irrelevant. (both before and now)
The call to `MavenTestingUtils.getTestResourcePath(String)` always returns an absolute path.

**Review by joakime — APPROVED** [review]
I can see a small change for the ee9 code to make it more stable for the ee9 -> ee8 translation process, but otherwise I'm happy with it.

**Review by joakime — APPROVED** [review]
Looking good.

**Review by olamy — APPROVED** [review]
and TCK pass now

**Pull request merged**